src/ansys/geometry/core/modeler.py

```diff
@@ -497,3 +499,9 @@ def prepare_tools(self) -> PrepareTools:
     def measurement_tools(self) -> MeasurementTools:
         """Access to measurement tools."""
         return self._measurement_tools
+
+    @property
+    @min_backend_version(25, 1, 0)
```

Comment:
Since you already have the decorator at method level (which I think it's better - let's us grow more organically) I would remove it from here.